containers.md

```diff
@@ -29,7 +29,7 @@ We have compiled a list of popular software within the container ecosystem that
 
 | Name                      | URL                           | Comment                |
 | :-----                    |:-----                         | :-----                 |
-| Istio	| https://github.com/istio/istio/releases/	| 1) arm64 binaries as of 1.6.x release series<br>2) [Istio container build instructions](https://github.com/aws/aws-graviton-getting-started/blob/main/containers-workarounds.md#Istio)|
+| Istio	| https://github.com/istio/istio/releases/	| 1) [Arm64 support is now available via the master branch](https://github.com/istio/istio/issues/26652). To try it out follow: https://github.com/istio/istio/wiki/Dev-Builds#dev-release-information. Support will be included in the upcoming 1.15 release ETA ~1 month.<br>2) [Istio container build instructions](https://github.com/aws/aws-graviton-getting-started/blob/main/containers-workarounds.md#Istio)|
```

Comment:
@JeffUnderhill could we provide absolute date and not relative (~ 1month).        alternatively, we can say "within few weeks" and update when it comes